### ISSUE TITLE
Add all contributors

### ISSUE THREAD

**Issue by goanpeca** [Napari/napari#3420]
## 🧰 Task

@neuromusic posted this on zulip (@melissawm also on slack)

Tracking here :) 

"GitHub - all-contributors/all-contributors: :sparkles: Recognize all contributors, not just the ones who push code :sparkles:" https://github.com/all-contributors/all-contributors

**Comment by goanpeca** [comment]
@sofroniewn we could use this bot to help https://allcontributors.org/docs/en/bot/installation :)

**Comment by sofroniewn** [comment]
Great! Yes, I love it - there was lots of support for this on the zulip, including from @jni so i think we're good to move forward with this

**Comment by liaprins-czi** [comment]
Would we want to have the additional contributors listed separately from the authors? Or have them all together and relabel the section from Authors to Contributors?

Here are a couple of ways this could look:
https://www.figma.com/file/pDKU0CdSzpSWGaoXXQVJLz/?node-id=1337%3A92

**Comment by sofroniewn** [comment]
> Would we want to have the additional contributors listed separately from the authors? Or have them all together and relabel the section from Authors to Contributors?

Are you thinking in terms of the napari hub here and the plugin page? I think there's also a need for the main repo too. I guess the tool above can generate tables like https://github.com/all-contributors/all-contributors#contributors-

Maybe that's just a standalone page on the napari.org website in the community section? cc @melissawm

**Comment by melissawm** [comment]
I think there are two ways to go about this, one is to include the table in the readme in the repo itself, the other is to include a page under the Community header for the napari.org website. It could be something like

- Mission and Values
- About the Project and Team
- Code of Conduct
- Handling Code of Conduct Reports
- Governance Model
- Translations
- Contributors <- all contributors table

edit: I'd also include a header for that page justifying "napari uses the all contributors model for both code and website/documentation content, as well as plugin development", or something like that - what do you think?

**Comment by liaprins-czi** [comment]
> Are you thinking in terms of the napari hub here and the plugin page? I think there's also a need for the main repo too. I guess the tool above can generate tables like https://github.com/all-contributors/all-contributors#contributors-

@sofroniewn Yes, I got confused and thought that this was referring to the plugins on the hub, and that that was why I was being mentioned in this issue.

**Comment by brisvag** [comment]
Would be nice to pick this up again :)
